src/utils/api.js

```diff
@@ -21,3 +21,9 @@ export function getAllArticles() {
     return data;
   });
 }
+
+export function getComments(article_id) {
+  return newsApi.get(`/articles/${article_id}/comments`).then(({ data }) => {
+    return data;
+  });
```

Comment:
Nice, but instead of sending down the data object and de-structuring on the other side, you can directly pass the specific part of this data that you actually want on the other side.